Managers/SettingsMan.cpp

```diff
@@ -144,17 +144,17 @@ int SettingsMan::ReadProperty(std::string propName, Reader &reader)
         g_FrameMan.ReadProperty(propName, reader);
     else if (propName == "NxFullscreen")
         g_FrameMan.ReadProperty(propName, reader);
-    else if (propName == "PostProcessing")
-        g_FrameMan.ReadProperty(propName, reader);
-    else if (propName == "PostPixelGlow")
-        g_FrameMan.ReadProperty(propName, reader);
     else if (propName == "PixelsPerMeter")
         g_FrameMan.ReadProperty(propName, reader);
     else if (propName == "PlayIntro")
         reader >> m_PlayIntro;
     else if (propName == "ToolTips")
         reader >> m_ToolTips;
-    else if (propName == "FlashOnBrainDamage")
+	else if (propName == "AdvancedPerformanceStats") {
+		bool enableAdvanced;
```

Comment:
I wonder if you can oneline this using reader.ReadValue()? Maybe not cause of the conversion (and safety checking, etc.) from string to bool being a pain.